db/migrate/20210209154429_change_setting_id_to_uuid.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeSettingIdToUuid < ActiveRecord::Migration[6.1]
+  def change
+    add_column :settings, :uuid, :uuid, default: "gen_random_uuid()", null: false
+
+    change_table :settings do |t|
+      t.remove :id
+      t.rename :uuid, :id
```

Comment:
is this the only table this happens?

Reply:
It's the only table that we're adding audits to that doesn't already have a UUID ID type:

```
Searching 34 files for "create_table([^,]+)do" (regex)

/home/chris/code/centauri/eyedp/db/migrate/20180208061509_create_friendly_id_slugs.rb:
    1  class CreateFriendlyIdSlugs < ActiveRecord::Migration[5.2]
    2    def change
    3:     create_table :friendly_id_slugs do |t|
    4        t.string   :slug,           :null => false
    5        t.integer  :sluggable_id,   :null => false

/home/chris/code/centauri/eyedp/db/migrate/20190525195835_create_settings.rb:
    1  class CreateSettings < ActiveRecord::Migration[5.2]
    2    def self.up
    3:     create_table :settings do |t|
    4        t.string  :var,        null: false
    5        t.text    :value,      null: true

/home/chris/code/centauri/eyedp/db/migrate/20190608181302_create_doorkeeper_tables.rb:
   17      add_index :oauth_applications, :uid, unique: true
   18  
   19:     create_table :oauth_access_grants do |t|
   20        t.references :resource_owner, index: true, type: :uuid
   21        t.references :application,     null: false, type: :uuid
   ..
   35      )
   36  
   37:     create_table :oauth_access_tokens do |t|
   38        t.references :resource_owner, index: true, type: :uuid
   39        t.references :application,    null: false, type: :uuid

/home/chris/code/centauri/eyedp/db/migrate/20190608181303_create_doorkeeper_openid_connect_tables.rb:
    1  class CreateDoorkeeperOpenidConnectTables < ActiveRecord::Migration[5.2]
    2    def change
    3:     create_table :oauth_openid_requests do |t|
    4        t.integer :access_grant_id, null: false
    5        t.string :nonce, null: false

/home/chris/code/centauri/eyedp/db/migrate/20190609183515_create_fido_usf_devices.rb:
    1  class CreateFidoUsfDevices < ActiveRecord::Migration[5.2]
    2    def change
    3:     create_table :fido_usf_devices do |t|
    4        t.references  :user,                  null: false, polymorphic: true, index: true, type: :uuid
    5        t.string      :name,                  null: false, default: ''

6 matches across 5 files
```

All of the other migrations actually define the primary key in the `create_table` call, like:

```ruby
create_table :groups, id: :uuid do |t|
      t.references :parent, index: true, type: :uuid, references: :groups
      t.text :name

      t.timestamps
    end
```